tests/test_parser.py

```diff
@@ -142,6 +142,47 @@ def test_open_json_file_raise_expected_exception_when_invalid_json_in_file(self)
             self.assertEqual(
                 'filename is not valid JSON.', str(context.exception))
 
+    def test_right_keys_format(self):
```

Comment:
please rename it to something like `test_validate_object_keys_returns_true_on_valid_params`